docs/topics/windows-vhd.md

```diff
@@ -0,0 +1,53 @@
+
+## Build Process
+
+Windows AKS VHDs are produced using [packer](http://www.packer.io) in an [Azure DevOps](http://dev.azure.com) build pipeline.
+
+The build pipeline:
+- Runs a packer job which provisions a new VM in Azure based off a specified Windows Server image and:
+  - Installs/enables required Windows components/features
+  - Download multiple versions of components which users are allowed to specify the version of (Kubernetes, Docker, etc)
+  - Downloads commonly used container images
+  - Configure system settings (windows update, page file sizes, etc)
+- Runs E2E tests using the VHD produced in the same pipeline
+- Optionally copies the VHD to another Azure storage account for extra validation and/or publishing 
+
+Note: See scripts referenced below to for further details
+
+### Build Resources
+
+- Azure DevOps build yaml build definition: [.pipelines/vhd-builder-windows.yaml](../../.pipelines/vhd-builder-windows.yaml)
+- Packer job definition: [packer/windows-vhd-builder.json](../../packer/windows-vhd-builder.json)
+- Customization script used in packer: [packer/configure-windows-vhd.ps1](../../packer/configure-windows-vhd.ps1)
+
+### Testing Private Fixes
+
+Occasionally it may be necessary to validate private fixes provided by the Windows team. This can be done without providing public access to the fixes by following these steps:
+- Fork Azure/aks-engine into a private git repo hosted in [Azure DevOps](http://dev.azure.com)
+- Configure a private build pipeline using [.pipelines/vhd-builder-windows.yaml](../../.pipelines/vhd-builder-windows.yaml) as the pipeline definition
+- Set up the build pipeline variables to point to private Azure subscriptions
+  - Pipeline variables are documented in the yaml file
+- Update [packer/configure-windows-vhd.ps1](../../packer/configure-windows-vhd.ps1) to install privates
+- Run the pipeline
+
+## Usage in aks-engine
+
+Both Windows and Linux nodes are configured by executing Custom Script Extensions as part of VM deployment operations.
+The scripts are generated by aks-engine which populates instance specific information from the [cluster definition API model](clusterdefinitions.md) and get embedded into the ARM templates produced by AKS engine. These scripts are enlightened to understand the work in conjunction with Windows AKS marketplace images to not duplicate configuration steps and/or utilize files cached during the VHD build process.
+
+[Parts/k8s/kuberneteswindowssetup.ps1](../../parts/k8s/kuberneteswindowssetup.ps1) and associated ps1 files are used as templates for the extension scripts.
+
+It is an explicit goal to maintain aks-engine compatibility with 'vanilla' Windows Server marketplace images (for most scenarios) and to treat the Windows AKS marketplace images as an optimization for the reasons stated above.

```

Comment:
It's probably worth calling out that installing patches on the 'vanilla' images is a non-goal.

Reply:
also rather than using the term 'vanilla' can you just add a reference to the actual marketplace image we're talking about? that should translate better :)